capture-sdk/sdk/src/doc/source/features.rst

```diff
@@ -381,6 +383,18 @@ You can show your own help screens in the Gini Capture SDK. You can pass the tit
 You can also disable the supported formats help screen by passing ``false`` to
 ``GiniCapture.Builder.setSupportedFormatsHelpScreenEnabled()``.
 
+Furthermore you can set back navigation action on a custom view on bottom navigation bar. You can pass your custom ``HelpNavigationBarBottomAdapter`` implementation to
```

Comment:
[Suggestion] When I read `set back navigation action` then I think I can set an action to be triggered on back navigation. Whereas in reality I can show a button to which I have to add the back action provided by the SDK. 

What do you think about using `show back navigation button` instead: `Furthermoe you can show a back navigation button on a custom view ...`